.golangci.reference.yml

```diff
@@ -107,6 +107,18 @@ output:
 
 # All available settings of specific linters.
 linters-settings:
+  asasalint:
+    # Add FuncName to exclude for some bad case report.
+    # Default: []
```

Comment:
I don't think this is the real default value.

Reply:
the default exclude inside the repo
```
Printf,Println,Errorf,Fprintf,Fprintln,Fatal,Fatalf,Panic,Panicf,Panicln,Print,Printf,Println,Sprintf,Sprintln,Error,Errorf,Info,Infof,Warn,Warnf,Debug,Debugf
```
this is too long and too complex for user to set, the `exclude` option here  is most like `extra-exclude`.

or  use `extra-exclude` as the config field name?

Reply:
you have to define the real default value even if it's long.

Reply:
In all cases, you will have to provide explicitly the default value of the exclude.
The user will want to know what is excluded or not.

Reply:
yes, i know you want to allow user know exactly which was `exclude`, not left user be confused why some error case are not report just because the default exclude.

and i rename the `default-exclude` to `builtin-exclude`, and add long note in `golangci.reference.yml` to explain how to config the `exclude` option, and how to close the `builtin-exclude`.

so user can use this two option to get what they really want to exclude, and no need add a long exclude option in the yaml like:

```
exclude:
 - Print
 - Printf
 - Println
 - Sprint
 - Sprintf
 - Sprintln
 - Fprint
 - Fprintf
 - Fprintln
 - ...
```